python/test/unit/operators/test_matmul.py

```diff
@@ -171,6 +175,9 @@ def init_input(m, n, dtype, acc_dtype):
         ret = (2.**exponents).to(getattr(torch, dtype)).to("cuda")
         return ret
 
+    if is_hip():
+        if INPUT_PRECISION == 'tf32x3' or is_fp8(ADTYPE) or is_fp8(BDTYPE):
+            pytest.skip("Skip fp8 inputs and tf32x3 precison on hip")
```

Comment:
Can we be more explicit in the message why it's skipped? Also it's `or`, not `and`. So something like "fp8 inputs or tf32x3 precision does not have native support on hip" or ".. unimplemented". This way it's clear to whoever running the test to know whether it will never be supported, or just something we need to fix in the future.